src/constants/game/CustomMapExperience.java

```diff
@@ -0,0 +1,21 @@
+package constants.game;
+
+import java.util.HashMap;
+
+public final class CustomMapExperience
+{
+    //Applies experience multiplier to MapID
+    public static final HashMap<Integer, Integer> CUSTOM_MAP_EXPERIENCE = new HashMap<Integer, Integer>() {{
+        put(50000, 20);
+    }};
+
+    public static boolean HasCustomExpValue(int mapId)

```

Comment:
this shouldn't be public. a class calling this shouldn't need to check if the map has a custom value, they should just ask for the multiplier, this class should check to see if it exists and if it doesn't it returns a 1